src/Sake.Engine/Logging/TeamCityLog.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+
+namespace Sake.Engine.Logging
+{
+    public class TeamCityLog : ILog
```

Comment:
Move this to a `_use-teamcity.shade` file.

for example
```
functions
  @{
    class TeamCityLog : Sake.Engine.Logging.ILog
    {
        public void Info(object value) { }
        public void Warn(object value) { }
        public void Verbose(object value) { }
        public void Error(object value) { }
    }
  }

@{
  // add correct "if" around this
  Log = new TeamCityLog(Settings);
}
```

Also add an `ISettings Settings {get;}` to `BuilderBase`